036_fibonacciSeries.js

```diff
@@ -0,0 +1,16 @@
+// 36. Fibonacci series up to n
+const fibonacci = (number) => {
```

Comment:
Fix logic 
Fibonacci upto 1000 should return only 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987
Your code is returning 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597